package_builder/build_claw.sh

```diff
@@ -101,29 +104,29 @@ if [[ ! -f $claw_compiler_install/libexec/claw_f_lib.sh || $REBUILD == YES ]]; t
   # Build claw-compiler dependency apache-ant
   echo "=============================="
   echo "Build claw-compiler dependency: apache-ant"
-  if [ ! -d ${package_basedir}/hpc-scripts ] ; then
-    git clone git@github.com:clementval/hpc-scripts.git ${package_basedir}/hpc-scripts
```

Comment:
don't rely on clementval user anymore, just copied the required script in buildenv